lumigator/python/mzai/backend/backend/api/routes/health.py

```diff
@@ -46,9 +51,24 @@ def get_job_metadata(job_id: UUID) -> JobSubmissionResponse:
             detail="Invalid JSON response"
         ) from e
 
+@router.get("/jobs/{job_id}/logs")
+def get_job_logs(job_id: UUID) -> JobLogsResponse:
+    resp = requests.get(ray_jobs() + f"{job_id}/logs")
+    if resp.status_code == 200:
+        try:
+            metadata = json.loads(resp.text)
+            return JobLogsResponse(**metadata)
+        except json.JSONDecodeError as e:
+            print(f"JSON decode error: {e}")
+            print(f"Response text: {resp.text}")
+            return {"error": "Invalid JSON response"}
+    else:
+        return {"error": f"HTTP error {resp.status_code}"}
```

Comment:
It might be useful to take a look at this recently merged PR to see how we have handled 200/400/{other}, so we don't just return `200 - OK` with some JSON containing an error.

https://github.com/mozilla-ai/lumigator/pull/318/files#diff-bc21eb24ef7e8d2c97660608a2ca09d7f46e7f6ffd2f749c4c75e03ad8bb20e8R25-R47

Reply:
BTW, let's keep an eye on https://docs.ray.io/en/latest/cluster/running-applications/job-submission/api.html?_gl=1*19sigrd*_up*MQ..*_ga*MTMzMzI3MjIxMS4xNzMxNDAxMjQ4*_ga_0LCWHW1N3S*MTczMTQwMTI0OC4xLjAuMTczMTQwMTI0OC4wLjAuMA.. 
I'll use that as reference for possible status codes.